assembl/static2/js/app/components/debate/thread/answerForm.jsx

```diff
@@ -154,7 +153,6 @@ export class DumbAnswerForm extends React.PureComponent<Props, State> {
                 <RichTextEditor
                   editorState={this.state.body}
                   handleInputFocus={this.handleInputFocus}
-                  maxLength={BODY_MAX_LENGTH}
```

Comment:
Is this required for Bright Mirror and/or admin views as well? If so, then you should remove the logic of `maxLength` in RichTextEditor class.

Reply:
Not sure if it is part of the scope of the bug or not

Reply:
I think it's just the thread

Reply:
Update: it's everywhere :--) will remove